lib/ontologies_linked_data/concerns/connectors/cordis_connector.rb

```diff
@@ -0,0 +1,178 @@
+require 'rexml/document'
+
+module Connectors
+  class CordisConnector < BaseConnector
+    private
+
+    def build_url
+      if params[:id]
+        base_url = connector_config[:base_url] 
+        "#{base_url}/#{params[:id]}"
+      else
+        connector_config[:search_url]
+      end
+    end
+
+    def build_params(params)
```

Comment:
if possible her it will better if the function has named arguments, instead of `params`, like this `build_params(project_id, project_acronym, limit, page)`